uberfire-rest/uberfire-rest-backend/src/main/java/org/guvnor/rest/backend/cmd/InstallProjectCmd.java

```diff
@@ -43,12 +43,14 @@ public JobResult internalExecute(JobRequest request) throws Exception {
         try {
             result = helper.installProject(jobRequest.getJobId(),
                                            jobRequest.getSpaceName(),
-                                           jobRequest.getProjectName());
+                                           jobRequest.getProjectName(),
+                                           jobRequest.getBranchName());
         } finally {
             JobStatus status = result != null ? result.getStatus() : JobStatus.SERVER_ERROR;
-            logger.debug("-----installProject--- , ouName: {}, project name: {} [{}]",
+            logger.debug("-----installProject--- , ouName: {}, project name: {}, branch name: {} [{}]",
```

Comment:
Do we want to use "ouName" in the logs? Maybe this could be finally renamed to "space" as "ouName" might not be understandable to users.